src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -367,7 +372,37 @@ public String generateLibraryContentLine(String cqlLibraryName, Version version)
     return "library " + cqlLibraryName + " version " + "'" + version + "'";
   }
 
-  private void saveMeasureBundle(Measure savedMeasure, String measureBundle, String username) {
+  public Export saveExport(
+      Measure savedMeasure,
+      String measureBundle,
+      String measureBundleWithoutWarnings,
+      String humanReadableWithCss) {
+    ObjectId measureBundleId =
+        mongoGridFsOperations.store(
+            new ByteArrayInputStream(measureBundle.getBytes()),
+            "measureBundle.json",
+            "application/json");
+    ObjectId measureBundleWithoutWarningsId =
+        mongoGridFsOperations.store(
+            new ByteArrayInputStream(measureBundleWithoutWarnings.getBytes()),
+            "measureBundleWithoutWarnings.json",
+            "application/json");
```

Comment:
I think this belongs to GridFsService- gridfs logic in GridFsService

Reply:
I went back and forth on this. I felt the gridfs `.store()` is similar to our repository `.save()`calls that we do throughout this class. Retrieving from gridfs is more complex than our repository `.find*()` calls, so to make it continue to match the existing repository usage pattern, we moved the retrieve work to a service and called it `find`.

Reply:
makes sense